examples/globe.html

```diff
@@ -51,6 +72,26 @@
             /* global itowns, document, GuiTools, globeView, promises */
             var menuGlobe = new GuiTools('menuDiv');
             menuGlobe.view = globeView;
+            var divScaleWidget = document.querySelectorAll('.divScaleWidget')[0];
+
+            function updateScaleWidget() {
+                var pitch = 0.28;
+                var size = 200 * pitch;
+                var scaled = globeView.controls.getScale();
+                var mm = size / scaled / 1000;
+                mm = Math.floor(mm);
+                var digit = Math.pow(10, mm.toString().length - 1);
+                var mmr = Math.round(mm / digit) * digit;
+                var pix = Math.floor(mmr * scaled / pitch * 1000);
```

Comment:
IHMO all this code indicates that `getScale` is a poor API.
Maybe you could improve it based on this experience, e.g:
```js
var m = globeView.controls.pixelsToSI(200);
// m.value = 1.23 and m.unit = 'km'
```